Improve the API documentation, Module MD, and Package MD (#179)
ballerina-platform/module-ballerina-io: io-ballerina/Package.md

## Package Overview

> The I/O library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina

Reviewer: ```suggestion

The I/O library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina
```

## Package Overview

The I/O library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina
</a> language.

This package provides APIs for file input/output operations and the console input/output.

> For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/learn/by-example/).

Reviewer: ```suggestion
```

### Report Issues

> To report bugs, request new features, start new discussions, view project boards, etc. please visit Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).

Reviewer 2: ```suggestion
To report bugs, request new features, start new discussions, view project boards, etc., go to the Ballerina Standard Library [parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
```
